identity/app/actions/AuthenticatedActions.scala

```diff
@@ -153,7 +153,7 @@ class AuthenticatedActions(
     authAction andThen recentlyAuthenticatedRefiner andThen apiVerifiedUserRefiner
 
   def authWithConsentRedirectAction: ActionBuilder[AuthRequest, AnyContent] =
-    recentlyAuthenticated andThen apiUserShouldRepermissionRefiner
+    noOpActionBuilder andThen permissionRefiner andThen apiUserShouldRepermissionRefiner
```

Comment:
Looks like this will still apply a recentlyAuthenticated check, can we test that?

Reply:
just tested it locally... It doesn't.